client/client.go

```diff
@@ -100,6 +101,29 @@ func (c *Client) handleEngineEvents() {
 
 // Begin API
 
+// Version returns the go-nitro version
+func (c *Client) Version() string {
```

Comment:
I wonder if we even need to bother with a `Version`  API anymore. If you're not using a built executable it returns `(devel)` which is not super helpful.  The `GetAddress` API call can be used instead of `Version` if we want some kind of easy sanity check. Similar to `Version` `GetAddress` also requires no parameters and it just returns a string.